src/app/fdctl/run/tiles/fd_replay.c

```diff
@@ -1062,6 +1123,31 @@ after_frag( fd_replay_tile_ctx_t * ctx,
         reset_fork = new_reset_fork;
       }
 
+      /* Update the gui */
+
+      if( ctx->replay_plugin_out_mem ) {
+        uchar msg[4098*8] __attribute__( ( aligned( 8U ) ) );
+        fd_memset( msg, 0, sizeof(msg) );
+        fd_blockstore_start_read( ctx->blockstore );
```

Comment:
I'm not familiar with the design here, but does it make sense to take a lock and do random reads from the blockstore to get the list of active (unrooted) slots on the current fork? This definitely should be tracked by replay tile on its own. cc @lidatong

Reply:
parent slot is already available via `slot_bank.prev_slot` so it doesn't actually need the blockstore here

Reply:
@lidatong is the full chain available?

Reply:
(You only need to report parents up until and including the first rooted parent)

Reply:
i added a smr check